src/sass/_list-view.scss

```diff
@@ -0,0 +1,201 @@
+@use '_variables.scss' as *;
+
+.list-container {
+	display: grid;
+	place-items: center;
+	.first-line {
+		font-family: $header-font;
+		color: $light-green;
+		font-weight: 400;
+		font-size: 60px;
+		line-height: 72px;
+		text-align: center;
+	}
+	.second-line {
+		font-family: $body-font;
+		font-style: normal;
+		font-weight: 400px;
+		font-size: 40px;
+		line-height: 60px;
+		text-align: center;
+		color: #ff404a;
+	}
+
+	.add-button {
+		position: relative;
+		transition: 0.5s ease-out;
+		width: 153px;
+		height: 114px;
+		background: $yellow;
+		border-radius: 40px;
+		font-family: $header-font;
+		font-style: normal;
+		font-weight: 400;
+		font-size: 30px;
+		line-height: 36px;
+		text-align: center;
+		color: $brown;
+		&::before {
+			position: absolute;
+			bottom: -37px;
+			left: -36px;
+			content: url('/img/before.svg');
+		}
+		&::after {
+			position: absolute;
+			top: -9px;
+			right: -38px;
+			content: url('/img/after.svg');
+		}
+	}
+}
+
+//list search view (with token) list.js
+
+.item-list-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+
+	form {
+		display: flex;
+		flex-direction: column;
+		align-items: center;
+
+		label {
```

Comment:
the `label` & `input[type='text']` can also be deleted, as they are covered in the font partial :)

Reply:
When I deleted them, many of the layouts changed. So I kept them.